### PR TITLE
3 - add switcher component

### PR DESCRIPTION
**Pull request by vitalyacode**

### CODE REVIEW

File: src/components/Unknown/Switcher/SwitcherExample.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+import Switcher from '.';
+
+const SwitcherExample: React.FC = () => {
+  const [value, setValue] = useState<string>('AMD');
+  return (
+    <Switcher value={value} setValue={setValue} choices={['AMD', 'Nvidia']} />
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/SwitcherExample.tsx]: it will be better to rename setValue -> onSwitch

Comment (VHarastei) [inline src/components/Unknown/Switcher/SwitcherExample.tsx]: choices -> options

File: src/components/Unknown/Switcher/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { ToggleButton, ToggleButtonGroup } from '@mui/material';
+import React from 'react';
+import useStyles from './styles';
+
+type SwitcherProps = {
+  value: string;
+  setValue: CallableFunction;
+  choices: [string, string];
+};
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/index.tsx]: choices - string[]

File: src/components/Unknown/Switcher/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      value={value}
+      onChange={handleSelect}
+      exclusive
+    >
+      <ToggleButton className={styles.switcherButton} value={firstValue}>
+        {firstValue}
+      </ToggleButton>
+      <ToggleButton className={styles.switcherButton} value={secondValue}>
+        {secondValue}
+      </ToggleButton>
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/index.tsx]: better to map through array options

File: src/components/Unknown/Switcher/styles.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { Theme } from '@mui/material';
+import { createStyles, makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    switcherButton: {
+      minWidth: 100,
+      borderRadius: `${theme.shape.borderRadius}px !important`,
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/styles.ts]: why !important?

Comment (vitalyacode) [reply src/components/Unknown/Switcher/styles.ts]: this is the only way to override default border-radius of ToggleButton

File: src/components/Unknown/Switcher/index.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      onChange={handleSelect}
+      exclusive
+    >
+      {options.map((option) => {
+        return (
+          <ToggleButton className={styles.switcherButton} value={option}>
+            {option}
+          </ToggleButton>
+        );
+      })}
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/index.tsx]: better to use inline map

File: src/components/Unknown/Switcher/index.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <ToggleButtonGroup
+      className={styles.toggleButtonGroup}
+      value={value}
+      onChange={handleSelect}
+      exclusive
+    >
+      {options.map((option) => {
+        return (
+          <ToggleButton className={styles.switcherButton} value={option}>
```
Comment (hampachovsky) [inline src/components/Unknown/Switcher/index.tsx]: add key attribute

File: src/components/Unknown/Switcher/SwitcherExample.tsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react';
+import Switcher from '.';
```
Comment (VHarastei) [inline src/components/Unknown/Switcher/SwitcherExample.tsx:2]: +

**Review by hampachovsky — APPROVED** [review]

**Review by VHarastei — APPROVED** [review]

**Pull request merged**